lib/whitehall/publishing_api.rb

```diff
@@ -53,14 +53,11 @@ def self.publish_redirect(redirect)
 
   private
 
-    # Note: this method does not account for non-translatable models, e.g.
-    # PolicyGroup. Once we are pushing those to the Publishing API, this method
-    # will need updating to return just the English locale for those models.
     def self.locales_for(model_instance)
-      if (locales = model_instance.translated_locales).empty?
-        [:en]
-      else
+      if model_instance.respond_to?(:translated_locales) && (locales = model_instance.translated_locales).any?
```

Comment:
Minor point, but the local assignment makes it trickier to read, especially now with the double check in the conditional. Arguably more readable as:

```ruby
if model_instance.respond_to?(:translated_locales) && model_instance.translated_locales.any?
  model_instance.translated_locales
else
  [:en]
end
```

Not offensive enough to block a merge though.